Cargo.toml

```diff
@@ -56,6 +57,7 @@ log = { workspace = true }
 once_cell = { workspace = true }
 openssl = { workspace = true, optional = true }
 p384 = { workspace = true }
+pbjson-types = { workspace = true }
```

Comment:
Remove this crate, you can use the astarte-message-hub-proto re-export.
```suggestion
```

Reply:
Used the re exposed traits